content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,86 @@ names in the `train` array below:
 
 ## Plot templates (data series only)
 
-Users have the ability to change the way data-series plots are displayed by
-modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
-generating plots in the style that best fits the their needs. This keeps
-<abbr>DVC projects</abbr> programming language agnostic, as it's independent
-from user display configuration and visualization code.
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications
+to create plots from user data. A set of built-in _plot templates_ are included.
 
-Built-in _plot templates_ are stored in the `.dvc/plots/` directory. The default
-one is called `default.json`. It can be changed with the `--template` (`-t`)
-option of `dvc plots show` and `dvc plots diff`. For templates in the
-`.dvc/plots/` directory, the path and the json extension are not required: you
-can specify only the base name e.g. `--template scatter`.
+The `linear` template is the default. It can be changed with the `--template`
+(`-t`) option of `dvc plots show` and `dvc plots diff`. The argument provided to
+`--template` can be a (built-in) template name or a path to a
+[custom template].
+
+<admon type="tip">
+
+For templates stored in `.dvc/plots` (default location for custom templates),
+the path and the json extension are not required: you can specify only the
+base name, e.g. `--template scatter`.
+
+</admon>
 
 DVC has the following built-in plot templates:
 
-- `default` - linear plot
+- `linear` - basic linear plot including cursor interactivity (default)
+- `simple` - simplest linear template (not interactive); Good base to create
+  [custom templates].
```

Comment:
Not for this PR: Should we talk more about this interactivity? I'm not even sure what it is and it's not mentioned much (at all actually) in other `plots` refs.

Reply:
I don't think so - interactivity is just to make user experience better, but its nothing major:
[simple example](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JACyUAVhDYA7EABoQAEySYUqUMSSCGcCGgDaoJFCgMATqYCeaEAAZKAdjEAOAJz2ATGM-2AbADMrgCMXgCsiiAAZjSCcLJICHB2wqScUDpIFhAEpuZWUNaUmBDEURZw5egA7mwWANYQOKYpShCYcDh29lGYNMkdSd3oIf7hTl5hU-5eTpEAvgomZpY2PZSuk06BIfZOk57+HlGx8YnJqTTp0Fk5eauFxaXlSpXVIHWNza1RHV12EJ9Ab6dQIEYgMYTKZeGZeVy9JYrArrdCOdxhMRiXYuQLhQIE-ynOIJJIpdBpUS3QTZXL5NZFEplCpVOxfJotKBtED-CFeYGDMEQqGTaZjeGLZYgelPDbuYLhMThVz+ELhEKhbyRJRnUmXCnXKmZGn3GU2JmvEDvNn1Dm-dqdCGBAWg4aA8ai2Hik5I6WPVEOSjuNyBNz2cL4jVKwL8nUki7kkCUjJ3On+xkvFkfdk-Ll-R12MQuobg93QsWzeUgX1mopy8LuULzdxzcL2dyzZ1x85kq43Y20h4ojPMt6s2q23Pc3l2bUgfqCt2jD0wuHuJzVqW12xooP+Ft4+weEJOffzDfdvWJ5PUwfbi1Zm3fTnTgvoIlKBeu0vL8teoL2ECNbpjugYdi4riHtiTgYg2sYxPGvYGv2qZDgyzyjla46fJOL75gC6AXvOIIlsKK4VoE4absi6FypMXiBJMrjwv4QSwq4xI9vqSaGimJppsOGGWtaE7PvaPJviAHGfiRQplp6MyUe41YALpLMRmDxHYUQ1DQyj0GglH2Eo8A0GQWCGfYxkgMgjRoKAmDWDg140LIKTqQkUBsMorlkPZIAAB7+bEcCCModgzp+TmJgAjgwSCyP06j9KQwKaYmM7qTuoAhWFdjbn00V2HFCVJWohppVp6AFe0xrkqAABecAWGwaDRDoEBwAs6leYI9TBTQoXheg1pRc5diyGwCCuTo1bdUAA)

[interactive](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JACyUAVhDYA7EABoQAEySYUqUMSSCGcCGgDaoJFCgMATqYCeaEAAZKAdjEAOAJz2ATGM-2AbADMrgCMXgCsiiAAZjSCcLJICHB2wqScUDpIFhAEpuZWUNaUmBDEURZw5egA7mwWANYQOKYpShCYcDh29lGYNMkdSd3oIf7hTl5hU-5eTpEAvgomZpY2PZSuk06BIfZOk57+HlGx8YnJqTTp0Fk5eauFxaXlSpXVIHWNza1RHV12EJ9Ab6dQIEYgMYTKZeGZeVy9JYrArrdCOdxhMRiXYuQLhQIE-ynOIJJIpdBpUS3QTZXL5NZFEplCpVOxfJotKBtED-CFeYGDMEQqGTaZjeGLZYgelPDbuYLhMThVz+ELhEKhbyRJRnUmXCnXKmZGn3GU2JmvEDvNn1Dm-dqdCGBAWg4aA8ai2Hik5I6WPVEOSjuNyBNz2cL4jVKwL8nUki7kkCUjJ3On+xkvFkfdk-Ll-R12MQuobg93QsWzeUgX1mopy8LuULzdxzcL2dyzZ1x85kq43Y20h4ojPMt6s2q23Pc3l2bUgfqCt2jD0wuHuJzVqW12xooP+Ft4+weEJOffzDfdvWJ5PUwfbi1Zm3fTnTgvoIlKBeu0vL8teoL2ECNbpjugYdi4riHtiTgYg2sYxPGvYGv2qZDgyzyjla46fJOL75gC6AXvOIIlsKK4VoE4absi6FypMXiBJMrjwv4QSwq4xI9vqSaGimJppsOGGWtaE7PvaPJviAHGfiRQplp6MyUe41YALpLMRmDxHYUQ1DQyj0GglH2Eo8A0GQWCGfYxlJkg1hwBYRigAkUBsMoNCyGQaCgAAHl5CFwIIyh2DOn7WDgiYAI4MEgsj9Oo-SkMCmmJjO6k7qAsQBUF6Dbn0YWRdFsV0GohpJVpOUgX8xrkqAABe9lsGg0Q6BAcALOpLmCPUfmZYFdjWqF4V2LIbAIO5OjVupNJ2Q5qDGCAyCNFcsgpL6rXxFA-RyH5NJMAFfmYPlwXuWQWlKCt2SgmgmAWHoSjbegCBsAwrVsKQDlKM5rknQYc0gL5KmfeCh3DXI3JQPE2R2E9L1wM9mCTUoi0NHYOBsO5COfbILluR5flsJydDpSALmyG5W3yJoPIBXAm3sJTNl7YIUTaLo5IhOprN6Gg9jtQsalSjdMUQNE9QII5CGCJ0s2gOttMU6kSBM5NgM2TNEvIwdR3oLdZ0k2wXWzSAZBWLY6lfbjnlU75VO9dlEkAoNBUxXFJWJXzUoWydB1wN5CNU4dQ3oFFLvFQl3J23YuXqTbGU0FlwUFk7dgh0V8WlWlPXx31FWCXlQcgKnrvh4jav2Rr2QowH2vzr7mPSsIZAM-E0T18oNvhEoyg7vw4Tm9j3141TnXddXBcjWNiTM3GCc66yfNqYvCxAA) - hover with mouse over the plot

Reply:
OK. Are most templates interactive though? (`layer/mark` element in the spec, I think)

We're only mentioning that these 2 are/are not. Minor indeed